VERSION ""

NS_ :
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:

BO_ 2024 OBD2: 8 Vector__XXX
 SG_ S01PID0D_VehicleSpeed m1 : 63|8@0+ (1,0) [0|255] "km/h" Vector__XXX
 SG_ S01PID05_EngineCoolantTemp m2 : 55|8@0+ (1,-40) [-40|215] "degC" Vector__XXX
 SG_ S01PID4D_BatterySOH m3 : 47|8@0+ (0.392156862745098, 0) [0|100] "%" Vector__XXX
 SG_ S01PID4E_BatterySOC m4 : 39|8@0+ (0.392156862745098, 0) [0|100] "%" Vector__XXX
 SG_ S01PID2F_FuelTankLevel m47 : 31|8@0+ (0.392156862745098,0) [0|100] "%" Vector__XXX
 SG_ S01PID21_DistanceMILOn m33 : 15|16@0+ (1,0) [0|65535] "km" Vector__XXX
 SG_ S01PID04_CalcEngineLoad m4 : 7|8@0+ (0.392156862745098,0) [0|100] "%" Vector__XXX